bookkeeper-common/src/main/java/org/apache/bookkeeper/common/util/SafeRunnable.java

```diff
@@ -39,7 +40,7 @@ default void run() {
         }
     }
 
-    void safeRun();
+    void safeRun() throws ExecutionException, InterruptedException;
```

Comment:
please don't change safeRun this way.
it is a core utility, there is no need to change this here:in any case you can catch your exception and handle them properly. This code is generic and the caller could not react correctly to exceptions, so it is useless to say that the method may throw checked exceptions

Reply:
like this?
        boolean isEnable = false;
        try {
            isEnable = metadataDriver.isHealthCheckEnabled().get();
        } catch (InterruptedException e) {
            LOG.error("isHealthCheckEnabled throw InterruptedException", e);
        } catch (ExecutionException e) {
            LOG.error("isHealthCheckEnabled throw ExecutionException", e);
        }
        if (!isEnable) {
            LOG.info("Health checks is currently disabled!");
            return;
        }

Reply:
Also, I removed ExecutionException, InterruptedException on method checkForFaultyBookies and safeRun